modules/preview-react/radio/lib/RadioGroup.tsx

```diff
@@ -76,12 +76,16 @@ export const RadioGroup = createContainer(Flex)({
       transition="100ms box-shadow"
       marginX={`-${space.xs}`}
       width="fit-content"
-      style={{
-        boxShadow:
-          errorColors.outer !== errorColors.inner
-            ? `inset 0 0 0 1px ${errorColors.outer}, inset 0 0 0 3px ${errorColors.inner}`
-            : `inset 0 0 0 2px ${errorColors.inner}`,
-      }}
+      style={
+        error !== undefined
```

Comment:
```suggestion
        !!error
```

What if somebody put `false` as a value for error?

Reply:
or just `error` as you have ternary

Reply:
I think error can only be `ErrorType` or undefined

Reply:
so `ErrorType` is always true, so no need to check specific type

Reply:
ErrorType is an enum so it can equal `0` (when Error) or `1` (when Alert). 
TS enums are so misleading, ```!!ErrorType.Error === false```